server/metrics.go

```diff
@@ -88,6 +88,14 @@ var (
 			Name:      "tso",
 			Help:      "Counter of tso events",
 		}, []string{"type"})
+
+	leaderTickCounter = prometheus.NewCounter(
```

Comment:
seem we can use tsoCounter to identify the leader.

Reply:
tsoCounter does not work for rawkv.

Reply:
For every leader tick, we will update the TSO no matter we use txn KV or raw KV.
 https://github.com/pingcap/pd/pull/793/files#diff-2c671b5ee7007449d6115af14481769eR263

Reply:
You are right :)